axelor-project/src/main/java/com/axelor/apps/project/service/roadmap/SprintService.java

```diff
@@ -16,14 +16,20 @@
  * You should have received a copy of the GNU Affero General Public License
  * along with this program.  If not, see <https://www.gnu.org/licenses/>.
  */
-package com.axelor.apps.project.service;
+package com.axelor.apps.project.service.roadmap;
 
 import com.axelor.apps.project.db.Project;
 import com.axelor.apps.project.db.ProjectVersion;
+import com.axelor.apps.project.db.Sprint;
+import java.util.List;
 
 public interface SprintService {
 
   void generateBacklogSprint(Project project);
 
   void generateBacklogSprint(ProjectVersion projectVersion);
+
+  List<Sprint> getSprintToDisplay(Project project);
+
+  String getSprintIdsToExclude(List<Sprint> sprintList);
```

Comment:
these method should be moved in a separate service, they have nothing to do with the method to generateBacklogSprint.

Also they are made to be called one after the other. It would be simpler to only create a single method in the interface that return directly the right String.